tox.ini

```diff
@@ -103,3 +103,10 @@ deps=
 extras= all,tests
 commands=
     pytest compatibility_tests/ --remote-data
+
+[testenv:bandit]
+basepython= python3.8
+deps=
+    bandit
+commands=
+    bandit -lll -r -x asdf/tests,asdf/commands/tests,asdf/tags/core/tests,asdf/extern asdf
```

Comment:
What happens if `bandit -ll` and you report MEDIUM severity issues?  Are there a lot of those?

Reply:
This PR fixes or explains away all issues at every severity level, so this would pass at `-l` or `-ll`.